music21/stream/makeNotation.py

```diff
@@ -619,8 +622,6 @@ def makeMeasures(
     for sp in spannerBundleAccum:
         post.append(sp)
 
-    post.coreElementsChanged()
```

Comment:
Ah -- we should find a way to try putting coreInsert above again now that you've likely fixed the bug.  This is a very slow operation that does not need to be.

Reply:
We would have to call it on `m`, but since `m` is refreshed in every loop iteration, it would be impossible / trivial (take your pick).